tests/app/test_queue.py

```diff
@@ -203,6 +229,18 @@ def test_expire_inflights(self, redis, redis_queue):
             assert redis.llen(Buffer.IN_FLIGHT.inflight_name(receipt1, QNAME_SUFFIX)) == 0
             assert redis.get("not_inflight") == b"test"
 
+    def test_put_batch_saving_expiry_metric(self, redis, redis_queue, mocker):
+        with self.given_inbox_with_many_indexes(redis, redis_queue):
+            pbsem_mock = mocker.patch("app.queue.put_batch_saving_expiry_metric")
+            Buffer.INBOX.inbox_name(QNAME_SUFFIX)
+            redis.set("not_inflight", "test")
+            redis_queue.poll(10)
+            redis_queue.poll(10)
+            redis_queue.poll(10)
+            time.sleep(2)
+            redis_queue.expire_inflights()
+            assert pbsem_mock.assert_called_with(3) is None
```

Comment:
Do you think we could put the new tests in a different class such as ` TestRedisQueueMetrics`? As it does not test the core feature of the queue but metrics around it? No strong take on that; we'd leverage the classes to test different features.

Reply:
done :+1